src/routes/plans/[id]/+page.svelte

```diff
@@ -1,6 +1,11 @@
 <svelte:options immutable={true} />
 
 <script lang="ts">
+  import ActivityIcon from '@nasa-jpl/stellar/icons/svg/activity.svg?component';
+  import BracesAsteriskIcon from 'bootstrap-icons/icons/braces-asterisk.svg?component';
+  import CalendarIcon from 'bootstrap-icons/icons/calendar3.svg?component';
```

Comment:
I'd use '@nasa-jpl/stellar/icons/svg/calendar.svg?component' instead as that's what's used in the other dropdown menus to represent Scheduling

Reply:
Updated